### ISSUE TITLE
[feature] Track line additions/deletions in current file and update Results panel(s) appropriately

### ISSUE THREAD

**Issue by gene-pavlovsky** [fdorg/flashdevelop#1650]
It would be nice if the Results panel(s) could keep updated when adding/removing lines from a file they reference.

Use case:
1) Try to compile, get several errors in a single file
2) Go to the first error (result), let's say it is a missing import. Ctrl+Shift+1 and voila, import is added. Or e.g. that line is some old stuff that's not needed anymore, delete the line then.
3) Press F12 to go to the next result (within the same file), but the position it takes you is actually out of date already, since there were lines added/removed.

**Comment by Neverbirth** [comment]
I agree on this one. Of course there is nothing we can do with files that our not open in the application, but it's to be expected.
